Jenkinsfile

```diff
@@ -39,14 +39,6 @@ node('ubuntu-zion') {
       }
     }
   }
-
-  stage('Build') {
-    //OsTools.runSafe(this, 'scripts/bundle.sh')
-  }
-
-  stage('Archive') {
-      archiveArtifacts artifacts: archiveName, onlyIfSuccessful: true
-  }
```

Comment:
this whole file will be overhauled in the upcoming changes to the build process, but for now, these thad to go.